mmcv/runner/hooks/lr_updater.py

```diff
@@ -194,18 +194,15 @@ def get_lr(self, runner, base_lr):
         progress = runner.epoch if self.by_epoch else runner.iter
 
         if isinstance(self.step, int):
-            lr = base_lr * (self.gamma**(progress // self.step))
-            if self.min_lr is not None:
-                # clip to a minimum value
-                lr = max(lr, self.min_lr)
-            return lr
-
-        exp = len(self.step)
-        for i, s in enumerate(self.step):
-            if progress < s:
-                exp = i
-                break
-        lr = base_lr * self.gamma**exp
+            stage = progress // self.step
```

Comment:
we may keep using `exp` rather than `stage` for a more clear usage.

Reply:
Errr, Zaida recommended me to use `stage` because he thinks `exp` is not clear?

Reply:
Use `exp` as before.